doc/source/examples/technology_showcase_examples/techdemo-28/ex_28-tecfricstir.rst

```diff
@@ -168,7 +168,7 @@ shape tool, as shown in the following figure:
     mapdl.cyl4(0, 0, r1, 90, r1, 180, h)
     mapdl.cyl4(0, 0, r1, 180, r1, 270, h)
     mapdl.cyl4(0, 0, r1, 270, r1, 360, h)
-    mapdl.vglue(3, 4, 5, 6)
+    mapdl.vglue(3, 4, 5, 6);
```

Comment:
we probably don't need the ;

```suggestion
    mapdl.vglue(3, 4, 5, 6)
```

Reply:
We do, otherwise it will show the output of the command. I could use `mute=True` though